python/lsst/ts/standardscripts/maintel/m1m3/check_hardpoint.py

```diff
@@ -0,0 +1,124 @@
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["CheckHardpoint"]
+
+import asyncio
+
+import yaml
+from lsst.ts.observatory.control.maintel.mtcs import MTCS, MTCSUsages
+
+from lsst.ts import salobj
+
+
+class CheckHardpoint(salobj.BaseScript):
+    """Check M1M3 Individual hardpoint breakaway.
+
+    Parameters
+    ----------
+    index : `int`
+        Index of Script SAL component.
+
+    Notes
+    -----
+    **Checkpoints**
+
+    - "Testing hardpoint {id}.": Check hardpoint.
+    - "Hardpoint breakaway check complete.": Check complete.
+
+    """
+
+    def __init__(self, index, add_remotes: bool = True):
+        super().__init__(index=index, descr="Check M1M3 Hardpoint")
+
+        self.mtcs = MTCS(
+            domain=self.domain,
+            intended_usage=None if add_remotes else MTCSUsages.DryTest,
+            log=self.log,
+        )
+
+        for comp in self.mtcs.components_attr:
+            if comp != "mtm1m3":
+                setattr(self.mtcs.check, comp, False)
+
+        self.timeout_check = 60
+        self.timeout_std = 60
+
+        self.hardpoints = range(1, 7)
+
+    @classmethod
+    def get_schema(cls):
+        url = "https://github.com/lsst-ts/"
+        path = (
+            "ts_externalscripts/blob/main/python/lsst/ts/standardscripts/"
+            "maintel/m1m3/check_hardpoint.py"
+        )
+        schema_yaml = f"""
+        $schema: http://json-schema.org/draft-07/schema#
+        $id: {url}{path}
+        title: CheckHardpoint v1
+        description: Configuration for Maintel check hardpoint breakaway test SAL Script.
+        type: object
+        properties:
+            hardpoints:
+                description: Hardpoints to run the breakway test.
+                oneOf:
+                  - type: array
+                    minItems: 1
+                    items:
+                      type: number
+                      minimum: 1
+                      maximum: 6
+                  - type: string
+                    enum: ["all"]
+                default: "all"
+        additionalProperties: false
+        """
+        return yaml.safe_load(schema_yaml)
+
+    async def configure(self, config):
+        self.hardpoints = (
+            range(1, 7) if config.hardpoints == "all" else config.hardpoints
+        )
+
+    def set_metadata(self, metadata):
+        metadata.duration = self.timeout_check * 2 + self.timeout_std
+
+    async def run(self):
+        # Check that the MTCS is in the right state
+        await asyncio.gather(
+            self.mtcs.assert_all_enabled(), self.mtcs.assert_liveliness()
+        )
+
+        await self.mtcs.enter_m1m3_engineering_mode()
+
```

Comment:
Mirror needs to be parked for hardpoint test to proceed. Can you please call lower_m1m3 (or similar), which shall become available soon, before proceeding with the test?